DragAndDrop/src/main/res/layout/drag_and_drop_target_layout.xml

```diff
@@ -9,99 +9,113 @@
     android:id="@+id/drop_target_zone"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:padding="@dimen/drag_and_drop_very_small_padding">
+    android:background="@color/white">
+
+    <TextView
```

Comment:
Use AppcompatTextVew and AppCompatImageView everywhere, for consistency (or TextView, ImageView)

Reply:
Nice catch Andrei. The Android recommendation is to use TextView/ImageView in layouts when not creating custom views:

"AppCompatTextVew will automatically be used when you use TextView in your layouts and the top-level activity / dialog is provided by appcompat. You should only need to manually use this class when writing custom views."